packages/react/src/redux.ts

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as Sentry from '@sentry/browser';
+import * as Redux from 'redux';
+
+export interface SentryMiddlewareOptions {
```

Comment:
This is no longer using middleware right? We should probably change the options name then.